packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec

```diff
@@ -8,7 +8,7 @@
 
 Summary:    MaaS Discovery Plugin for Foreman
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    18.0.4
```

Comment:
you somehow managed to not to drop the old 18.0.4 tarball from git, which confuses the build system